tools/bumper-bot/bumper.go

```diff
@@ -0,0 +1,104 @@
+//usr/bin/env go run $0 "$@"; exit
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/kubevirt/cluster-network-addons-operator/tools/bumper-bot/parser"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Error: must contain path to components yaml", os.Args)
+		return
+	}
+	componentsYaml := os.Args[1]
+
+	fmt.Println("parsing", componentsYaml)
+	components, err := parser.ParseComponentsYaml(componentsYaml)
+	if err != nil {
+		fmt.Println(errors.Wrap(err, "failed to parse components yaml"))
+		return
+	}
+
+	for i, component := range components {
+		printCurrentComponentParams(component)
+		currentReleaseVersion, err := getCurrentReleaseTag(component.Url)
+		if err != nil {
+			fmt.Println(errors.Wrapf(err, "failed to get latest release version tag from %s", component.Url))
+			return
+		}
+
+		latestReleaseVersion, latestReleaseCommit, err := getLatestReleaseInfo(component.Url)
+		if err != nil {
+			fmt.Println(errors.Wrapf(err, "failed to get latest release version tag from %s", component.Name))
+			return
+		}
+
+		//TODO make updatePolicyDummy dynamic
```

Comment:
What does this mean? Would be this comment useful to anybody but you?

Reply:
This is just a placeholder that be be changed in future task

Reply:
Then it should be verbose enough that it would be clear what needs to be done on this place. Otherwise it is just a headscratching comment.